ibtk/Makefile.in

```diff
@@ -188,8 +188,7 @@ am__DIST_COMMON = $(srcdir)/Makefile.in \
 	$(top_srcdir)/config/install-sh $(top_srcdir)/config/ltmain.sh \
 	$(top_srcdir)/config/missing config/compile \
 	config/config.guess config/config.rpath config/config.sub \
-	config/depcomp config/install-sh config/ltmain.sh \
-	config/missing
+	config/install-sh config/ltmain.sh config/missing
```

Comment:
This isn't the most important problem we have but `depcomp` really should be installed. Are you sure that you are using autotools 2.69?